sklearn/base.py

```diff
@@ -516,13 +546,30 @@ def score(self, X, y=None):
         pass
 
 
-###############################################################################
 class MetaEstimatorMixin(object):
+    _required_parameters = ["estimator"]
+
     """Mixin class for all meta estimators in scikit-learn."""
```

Comment:
shouldn't the attribute go after the docstring?

Reply:
I don't know?